modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -1,17 +1,14 @@
-= Refining Batch Steps Processing
+= Using Batch Processing Scopes
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :page-aliases: batch-aggregator-concept.adoc, filter-records-batch-faq.adoc
 
-You can refine the work that a batch step performs upon the records it processes.
+You can set up and configure Mule batch processing scopes to perform common batch processing use cases and to enhance the performance of batch jobs for cases in which the defaults require adjustment.
```

Comment:
I think that we should stick to filter and aggregate here, that is, briefly state that you can configure the way a batch step will handle the records it gets from the stepping queue, allowing filtering and post processing aggregation.

Reply:
The issue here is that we are talking about more topics than filtering and aggregation on this page, so that info is misleading. So it doesn't make sense to mention only a subset of topics. And links to the topics will be visible in the right nav, so there is no reason to repeat them here. It's redundant.

Reply:
We can discuss this more, but I really need help with the technical issues mentioned below.